NEWS

```diff
@@ -1,8 +1,9 @@
 XLConnect News
 --------------
 1.0.8 tbd
- * avoid log4j2 error on first use (#172) 
- * update POI to 5.2.5
+ * Avoid log4j2 error on first use (#172) 
+ * Update POI to 5.2.5
```

Comment:
Probably should be aligned with the wording you corrected below.

Reply:
Done